src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -58,11 +58,11 @@
 import com.owncloud.android.R;
```

Comment:
Please include your name as @author and update the year at the top of this class

Reply:
The copyright year is not the current one yet